.github/workflows/build.yml

```diff
@@ -1,26 +1,39 @@
 name: build-ublue
 on:
+  # Build pull requests whenever they are opened or updated, to make sure they
+  # work. The build won't be deployed, since we filter out PRs in the deployment
+  # stage. PRs cannot bypass this, since only the repo's own workflow is used.
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
   pull_request:
     branches:
-      - main
+      - publish
```

Comment:
I kinda have a bad taste about calling it "`publish`" as that is a verb and branches are things (and even in verbs I'd prefer "`build`". Calling it `public` wouldn't suit either (all branches are public), but how about `custom` or smth?

Reply:
@EinoHR After consulting with ChatJippity for other name ideas, it rehashed the ones we've tried but then it came up with one more: `live`. It could fit. So that's the "live" (deployed, published, public) image.

- `template`: The template.
- `live`: The branch that goes live (deploys).

Alternatively:

- `template`: The template.
- `publish`: The branch that gets published (deploys).

We should probably continue this on Discord and ask for more feedback.

Reply:
We ended up agreeing on `live` being the best one of this bunch.